backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/course/CourseExecution.java

```diff
@@ -142,6 +146,10 @@ public void addAssessment(Assessment assessment) {
         assessments.add(assessment);
     }
 
+    public void addSubmission(Submission submission) {
+        submissions.add(submission);
+    }
+
```

Comment:
should approved questions be kept, eliminating only the associated submission/review? or should all submitted questions in course execution be eliminated? since questions are only associated with the course itself

Reply:
It is necessary to define when a submission can be deleted and who can delete it. The submission can be deleted only if its question can be deleted as well. I would say that it could not possible to delete a submission if the teacher has already started the interaction.